bindata/hypershift/ovn-kubernetes/alert-rules-control-plane.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
This file is a big TODO. Where is this monitored? Do we need to add a metrics port to the Route?

Reply:
added TODO for ovnkube-master monitoring in monitor.yaml file.